README.md

```diff
@@ -1,2 +1,62 @@
-# leaderboard
-Leaderboard app interacting with an API meant to practice async Javascript
+# Leaderboard
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+
+> Leaderboard app that interacts with an API for the purpose of practicing async Javascript
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+ko
+## Built With
+
+- HTML&CSS
+- Javascript
+- Webpack
+
+## Live Demo
+[Live Demo Link](#)
```

Comment:
- [ ] My comment on the screenshot applies here. Once we have a demo we can add it here.